README.md

```diff
@@ -0,0 +1,202 @@
+
+### Tech Stack <a name="tech-stack"></a>
+
+> HTML and CSS Languages
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://html.com/">HTML</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Styling</summary>
+  <ul>
+    <li><a href="https://css-tricks.com/">CSS</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> The key features added in this project.
+
+- Mobile Version
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> To get started as a deveoper, below is instructions on how to make use of this project.
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+1. Latest version of your browser
+2. Code Editor
+
+<!--
+Example command:
+
+```sh
+ gem install rails
+```
+ -->
+
+### Setup
+
+Clone this repository to your desired folder:
+
+Run this command
+
+```sh
+  cd Folder_Name
+  git clone git@github.com:manzitresor/Portfolio.git
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  cd Portfolio
+  npm install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+Open index.html using liver server extension.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> This is a solo project, successfully submitted by myself;
+
+👤 Manzi Tresor
+
+- GitHub: [@githubhandle](https://github.com/manzitresor)
+-Twitter: [@githubhandle](https://twitter.com/MANZITresor3)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/manzi-tresor-783b4022a/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Features to be added in this project include.
+
+- Home Page For mobile version
+- About Page for mobile vesrion
+- Desktop Version
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to Reach Out To me 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, please support me by following me on Github; <a href ="https://github.com/manzitresor/Hello_Microverse.git">Manzi</a> and give a star to this project.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank the Microverse team for making this program possible.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
+## ❓ FAQ <a name="faq"></a>
+
+> Frequently Asked Questions
+
+- Which browser works best for this project?
```

Comment:
- [ ] The previous reviewer asked for the MIT link to be fixed not removed. please kindly fix that. please always ensure to include all the main headers to your README and info under it. only optional one is the FAQ. you are missing the license section. kindly add it back and fix it. 
we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish.